css/css-flexbox/flexbox_first-letter.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<title>flexbox | first-letter</title>
+<meta name="assert" content="This test is ensures that flexbox placement does not apply to ::first-letter psuedo-content">
+<link rel="author" title="Vince Falconi" href="vince.falconi@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/css-flexbox-1/#placement">
+<style>
+  div { display: flex; }
+  div::first-letter { order: 2 }
+</style>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<div>Triceratops</div>
+
+<script>
+  test(function() {
+    let order = getComputedStyle(document.querySelector('div'), '::first-letter').order;
```

Comment:
Also, I am wondering if you considered using document.getElementById considering it would perform better than querySelector in this case. There is an interesting reddit link posted below where performance between these methods is debated.
https://www.reddit.com/r/learnjavascript/comments/356k1v/confused_on_queryselector_and_getelementbyid/

Reply:
Performance shouldn't really be a concern here (page load time will vastly dominate), and going for whatever's simplest makes as much sense as anything else.

Reply:
I agree with @gsnedders - to expound on it a bit, there area where you'll greatly benefit from getElementById vs querySelector is going to be on a very deep DOM tree. This is not the case here, so you're probably only going to save a few microseconds. Regarding `let` vs `const` you're technically correct regarding the semantics of the their existence but that has no implications with regard to how these tests are run. There is no potential namespace collisions as they occur in a vacuum - so `let` is fine.